packages/Organisms/SiteHeader/index.vue

```diff
@@ -332,9 +344,32 @@ export default {
       }
 
       img {
-        width: $rpl-site-header-logo-width;
+        width: 6.75rem;
         margin-left: $rpl-site-header-menu-toggle-border-spacing;
       }
+
+      &--vic-logo-static {
```

Comment:
Can you please rename this to something more general, like logo-primary ?

Reply:
Updated name to vic-logo-primary